tracer/src/Datadog.Trace/Logging/DirectSubmission/Formatting/LogFormatter.cs

```diff
@@ -29,6 +31,7 @@ internal class LogFormatter
         private readonly string? _globalTags;
```

Comment:
If I understand right, we should not use `_globalTags` anymore, but `_enrichedGlobalTags` instead.
Can we remove `_globalTags` field? Or at least add comment, that it shouldn't be used?

Reply:
Fixed as part of 52d6dff455366205f331e1c766c27cb86804cc07